.github/workflows/awsfulltest.yml

```diff
@@ -3,7 +3,7 @@ name: nf-core AWS full size tests
 # It can be additionally triggered manually with GitHub actions workflow dispatch button.
 # It runs the -profile 'test_full' on AWS batch
 
-on:
+on: # yamllint disable-line rule:truthy
```

Comment:
is this from tools@dev?

Reply:
No, that was me trying stuff out

Reply:
but I think actually I can remove all that out since I added the truthly thingy in the .yamllint